test/code_size/hello_world_wasm2js.js

```diff
@@ -35,33 +35,43 @@ function e(b) {
                     }
                 }),
                 c: function() {},
-                d: function(p, t) {
+                d: function(c, w) {
                     v(1024);
                     return 0;
                 },
-                e: c
+                e: f,
+                f: function(c) {
+                    n = c | 0;
+                },
+                g: function(c) {
+                    n = c = n - (c | 0) & -16;
+                    return c | 0;
+                },
+                h: function() {
+                    return n | 0;
+                }
```

Comment:
These three new functions are maybe worth looking into. The first and last should be getting inlined, surely.

Perhaps we can add a tiny Acorn pass to deduplicate the mangled/unmangled globals, if they are the cause of this, and if Closure is not good enough to do it itself.